br/Makefile

```diff
@@ -35,9 +39,9 @@ LDFLAGS += -X "github.com/tikv/migration/br/pkg/version/build.GitBranch=$(shell
 check: check/tidy check/golangci-lint check/gosec
 
 check/tidy:
-	cp go.sum /tmp/go.sum.origin
+	cp go.sum $(GO_SUM_TMPFILE)
```

Comment:
since `GO_SUM_TMPFILE` is only used in this build target, can we generate the tmp file name inside the process of this build target?